internal/pkg/cli/exec.go

```diff
@@ -15,4 +15,5 @@ type execVars struct {
 	taskID           string
 	containerName    string
 	skipConfirmation bool
+	yes              *bool
```

Comment:
I wonder if we should make this flag more descriptive, like `ssmPlugin` or `updateSSM` or something?

Reply:
Do we need the new `yes` field? (see below) I think we can get away by making this a pointer and assigning a local variable instead of storing this as a field in the struct
```suggestion
	skipConfirmation                      *bool  // If nil, we will prompt to upgrade the ssm plugin.
```

Reply:
ok done!